src/components/Button/Button.tsx

```diff
@@ -1,29 +1,58 @@
-import React from 'react';
+import React, { MouseEventHandler } from 'react';
+import classNames from 'classnames';
+
+import Spinner from '../utils/Spinner/Spinner';
```

Comment:
Spinner is not a util fn to me. I think we can also expose it in the docs

Reply:
You are right! I'll create a ticket for this